Index: config/BAG_GBA.xrv
===================================================================
--- config/BAG_GBA.xrv	(revision 33)
+++ config/BAG_GBA.xrv	(working copy)
@@ -3,12 +3,12 @@
   <!-- comment logging for output to commandline -->
   <logging>
     <database-provider>System.Data.OracleClient</database-provider>
-    <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=;</database-connection>
+    <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=wachtwoord;</database-connection>
   </logging>
   <sources>
     <source id="bag">
       <database-provider>System.Data.OracleClient</database-provider>
-      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=;</database-connection>
+      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=wachtwoord;</database-connection>
       <database-query>
 -- verblijfsobjecten --
 SELECT
@@ -122,7 +122,7 @@
     </source>
     <source id="gba">
       <database-provider>System.Data.OracleClient</database-provider>
-      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=;</database-connection>
+      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=wachtwoord;</database-connection>
       <database-query>
 SELECT 
   T1."BSN",
Index: config/BAG_KVK.xrv
===================================================================
--- config/BAG_KVK.xrv	(revision 32)
+++ config/BAG_KVK.xrv	(working copy)
@@ -3,12 +3,12 @@
   <!-- comment logging for output to commandline -->
   <logging>
     <database-provider>System.Data.OracleClient</database-provider>
-    <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=;</database-connection>
+    <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=wachtwoord;</database-connection>
   </logging>
   <sources>
     <source id="bag">      
       <database-provider>System.Data.OracleClient</database-provider>
-      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=;</database-connection>
+      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=wachtwoord;</database-connection>
       <database-query>
 -- verblijfsobjecten --
 SELECT
Index: config/BAG_KVK_ZONDERACCENT.xrv
===================================================================
--- config/BAG_KVK_ZONDERACCENT.xrv	(revision 33)
+++ config/BAG_KVK_ZONDERACCENT.xrv	(working copy)
@@ -3,12 +3,12 @@
   <!-- comment logging for output to commandline -->
   <logging>
     <database-provider>System.Data.OracleClient</database-provider>
-    <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=;</database-connection>
+    <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=wachtwoord;</database-connection>
   </logging>
   <sources>
     <source id="bag">      
       <database-provider>System.Data.OracleClient</database-provider>
-      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=;</database-connection>
+      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=wachtwoord;</database-connection>
       <database-query>
 -- verblijfsobjecten --
 SELECT
@@ -129,7 +129,7 @@
     </source>
     <source id="kvk">
       <database-provider>System.Data.OracleClient</database-provider>
-      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=;</database-connection>
+      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=wachtwoord;</database-connection>
       <database-query>
 SELECT DISTINCT kvkadres.*
 FROM (
Index: config/BAG_PBS.xrv
===================================================================
--- config/BAG_PBS.xrv	(revision 33)
+++ config/BAG_PBS.xrv	(working copy)
@@ -3,12 +3,12 @@
   <!-- comment logging for output to commandline -->
   <logging>
     <database-provider>System.Data.OracleClient</database-provider>
-    <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=;</database-connection>
+    <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=wachtwoord;</database-connection>
   </logging>
   <sources>
     <source id="bag">
       <database-provider>System.Data.OracleClient</database-provider>
-      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=;</database-connection>
+      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=wachtwoord;</database-connection>
       <database-query>
 -- verblijfsobjecten --
 SELECT
@@ -122,7 +122,7 @@
     </source>
     <source id="pbs">
       <database-provider>System.Data.OracleClient</database-provider>
-      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=;</database-connection>
+      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=wachtwoord;</database-connection>
       <database-query>
 		SELECT 
 			POSTKODE,
Index: config/BAG_PREVENT.xrv
===================================================================
--- config/BAG_PREVENT.xrv	(revision 32)
+++ config/BAG_PREVENT.xrv	(working copy)
@@ -3,12 +3,12 @@
   <!-- comment logging for output to commandline -->
   <logging>
     <database-provider>System.Data.OracleClient</database-provider>
-    <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=;</database-connection>
+    <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=wachtwoord;</database-connection>
   </logging>
   <sources>
     <source id="bag">      
       <database-provider>System.Data.OracleClient</database-provider>
-      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=;</database-connection>
+      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=wachtwoord;</database-connection>
       <database-query>
 -- verblijfsobjecten --
 SELECT
@@ -137,7 +137,7 @@
     </source>
     <source id="gba">
       <database-provider>System.Data.OracleClient</database-provider>
-      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=;</database-connection>
+      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=wachtwoord;</database-connection>
       <database-query>
 		  SELECT 
 			gba_tgbaadr.nopr,
@@ -155,7 +155,7 @@
     </source>
     <source id="prevent">
       <database-provider>System.Data.OracleClient</database-provider>
-      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=;</database-connection>
+      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=wachtwoord;</database-connection>
       <database-query>
 		SELECT *
 		FROM PREVENT.ADDRESS@PREV ADDRESS
Index: config/BAG_VERGUNNING.xrv
===================================================================
--- config/BAG_VERGUNNING.xrv	(revision 32)
+++ config/BAG_VERGUNNING.xrv	(working copy)
@@ -3,12 +3,12 @@
   <!-- comment logging for output to commandline -->
   <logging>
     <database-provider>System.Data.OracleClient</database-provider>
-    <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=;</database-connection>
+    <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=wachtwoord;</database-connection>
   </logging>
   <sources>
     <source id="bag">
       <database-provider>System.Data.OracleClient</database-provider>
-      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=;</database-connection>
+      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=wachtwoord;</database-connection>
       <database-query>
 -- verblijfsobjecten --
 SELECT
@@ -122,7 +122,7 @@
     </source>
     <source id="gba">
       <database-provider>System.Data.OracleClient</database-provider>
-      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=;</database-connection>
+      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=wachtwoord;</database-connection>
       <database-query>
 		  SELECT 
 			gba_tgbaadr.nopr,
@@ -138,7 +138,7 @@
     </source>
     <source id="vergunning">
       <database-provider>System.Data.OracleClient</database-provider>
-      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=;</database-connection>
+      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=wachtwoord;</database-connection>
       <database-query>
 		SELECT DISTINCT
 		  VGVOB."STRAAT" AS openbareruimtenaam,
Index: config/BAG_WOZ.xrv
===================================================================
--- config/BAG_WOZ.xrv	(revision 32)
+++ config/BAG_WOZ.xrv	(working copy)
@@ -3,12 +3,12 @@
   <!-- comment logging for output to commandline -->
   <logging>
     <database-provider>System.Data.OracleClient</database-provider>
-    <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=;</database-connection>
+    <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=wachtwoord;</database-connection>
   </logging>
   <sources>
     <source id="bag">
       <database-provider>System.Data.OracleClient</database-provider>
-      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=;</database-connection>
+      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=wachtwoord;</database-connection>
       <database-query>
 -- verblijfsobjecten --
 SELECT
@@ -121,7 +121,7 @@
     </source>
     <source id="woz">
       <database-provider>System.Data.OracleClient</database-provider>
-      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=;</database-connection>
+      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=wachtwoord;</database-connection>
 		<database-query>
 SELECT DISTINCT
   T1."IDENT" AS wozobjectid,
Index: config/DDS_NNP_K2P.xrv
===================================================================
--- config/DDS_NNP_K2P.xrv	(revision 32)
+++ config/DDS_NNP_K2P.xrv	(working copy)
@@ -3,7 +3,7 @@
   <!-- comment logging for output to commandline -->
   <logging>
     <database-provider>System.Data.OracleClient</database-provider>
-    <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=;</database-connection>
+    <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=wachtwoord;</database-connection>
   </logging>
   <sources>
     <source id="dds">
@@ -27,7 +27,7 @@
     </source>
     <source id="k2p">
       <database-provider>System.Data.OracleClient</database-provider>
-      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=;</database-connection>
+      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=wachtwoord;</database-connection>
       <database-query>
 SELECT   
   pvs_tvergunning.kgbr,
Index: config/DDS_NP_K2P.xrv
===================================================================
--- config/DDS_NP_K2P.xrv	(revision 32)
+++ config/DDS_NP_K2P.xrv	(working copy)
@@ -4,7 +4,7 @@
   <!--
   <logging>
     <database-provider>System.Data.OracleClient</database-provider>
-    <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=;</database-connection>
+    <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=wachtwoord;</database-connection>
   </logging>
   -->
   <sources>
@@ -31,7 +31,7 @@
     </source>
     <source id="k2p">
       <database-provider>System.Data.OracleClient</database-provider>
-      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=;</database-connection>
+      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=wachtwoord;</database-connection>
       <database-query>
 SELECT   
   pvs_tvergunning.kgbr,
Index: config/GBA_K2P.xrv
===================================================================
--- config/GBA_K2P.xrv	(revision 33)
+++ config/GBA_K2P.xrv	(working copy)
@@ -3,12 +3,12 @@
   <!-- comment logging for output to commandline -->
   <logging>
     <database-provider>System.Data.OracleClient</database-provider>
-    <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=;</database-connection>
+    <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=wachtwoord;</database-connection>
   </logging>
   <sources>
     <source id="gba">
       <database-provider>System.Data.OracleClient</database-provider>
-	  <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=;</database-connection>
+	  <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=wachtwoord;</database-connection>
       <database-query>
 SELECT 
   T1."BSN",
@@ -35,7 +35,7 @@
     </source>
     <source id="k2p">
       <database-provider>System.Data.OracleClient</database-provider>
-      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=;</database-connection>
+      <database-connection>Data Source=gegevensbeheer;User Id=gegevensbeheer;Password=wachtwoord;</database-connection>
       <database-query>
 SELECT
   pvs_tvergunning.kgbr,
